<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoECE</class>
 <widget class="QWidget" name="AutoECE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="pbAdvancedOptions">
     <property name="text">
      <string>Advanced Options</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="formAdvancedOptions" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbDilation">
        <property name="text">
         <string>Dilation radius</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="sbDilation">
        <property name="value">
         <double>1.800000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbFinalNSuperpixels">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Final number of superpixels (N_f &lt;&lt; N_0)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="sbFinalNSuperpixels">
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="value">
         <number>500</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Supervoxel Method</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="cBoxSupervoxel">
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>DISF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SICLE</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Segmentation Method</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="cBoxSegmentation">
        <item>
         <property name="text">
          <string>DynamicTrees</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Watershed</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbInitialNSuperpixels">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Initial number of seeds (N_0 &gt; 0)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="sbInitialNSuperpixels">
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="value">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="cBoxRun">
        <item>
         <property name="text">
          <string>All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dilatation</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Superspels</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Select Labels</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbTimeofImages">
     <property name="text">
      <string>Time of Images (s)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSlider" name="sliderTimeImages">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>7</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbImage">
       <property name="text">
        <string>Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbCurrentImageIndex">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="teImages">
       <property name="currentSection">
        <enum>QDateTimeEdit::MinuteSection</enum>
       </property>
       <property name="displayFormat">
        <string notr="true">mm:ss</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pbBackgroundSeeds">
     <property name="text">
      <string>Select Background Seeds</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Render Beautifully</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbRun">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbViewECECurves">
     <property name="text">
      <string>View ECE Curves</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
